modules/retainnumberofreports.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+"""
+This module is to define the retention period for reports
+"""
+
+import os
+import pathlib
+import glob
+import shutil
+import logging
+from modules import gcpaudit
+from modules import awsaudit
+from modules import azureaudit
```

Comment:
By importing the modules by default, will result in breaking of the tool

Let's assume the case, where a user is running audit for Azure environment
`from modules import awsaudit`
The awsaudit file will run this line
https://github.com/opstree/cs-suite/blob/master/modules/awsaudit.py#L30

Which basically requires AWS cli to be configured.
Hence breaking it, as the user will be running the scan only for Azure.
Can we import these based on the environment parameter given by the user

Something like this
https://github.com/opstree/cs-suite/blob/master/cs.py#L65

Reply:
Done with the suggested changes, please review